uptane-standard.md

```diff
@@ -803,9 +806,9 @@ In order to perform partial verification, an ECU SHALL perform the following ste
 
 Full verification of metadata means that the ECU checks that the Targets metadata about images from the Director repository matches the Targets metadata about the same images from the Image repository. This provides resilience to a key compromise in the system.
 
-Full verification MAY be performed by either primary or secondary ECUs. The procedure is the same, except that secondary ECUs receive their metadata from the primary instead of downloading it directly. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
+Full verification MAY be performed by either primary or secondary ECUs. In the following instructions, whenever an ECU is directed to download metadata, it applies only to primary ECUs.
```

Comment:
Why remove this sentence?

Reply:
Provided clarification on call, please let me know if additional clarity is needed.

Reply:
Can we get a final review on this so it can be merged?

Reply:
I missed the call, but if your goal is to obfuscate the language about downloading, it still reads "whenever an ECU is directed to download metadata". I don't see what the problem was with the removed sentence in any case; we haven't specified here *where* the metadata is downloaded from.

Reply:
Hi Patrick, thanks for your feedback, please allow me to clarify.
The discussion on the call concerning removing this line is due to the fact that it is interpreted to provide more confusion than clarification. Also, the removal of this line does not negatively impact nor remove meaning for the entire section so it seemed reasonable. Lastly, it was discussed that if an implementer would like to know the difference in the update process for primary-v-secondary, they can clearly see if detailed in the lists for each section.

Hope this helps!

Reply:
No argument there, but my primary concern wasn't addressed: if your goal is to obfuscate the language about downloading from an external network, it still reads "whenever an ECU is directed to download metadata". Do you consider sending metadata from a connected ECU to the primary to constitute "downloading"? If so, then perhaps there isn't a problem. But when I hear the word "download", I assume that means there is an external network connection.

Reply:
Thanks for the feedback, Patrick.

I understand your concern.
Personally, yes - I believe 'download' can be interpreted in a generic sense which does not require an external network connection. 
My recommendation would be, (in a future revision of Uptane, 1.1?), we can provide more clarification around specific terms used throughout the document.
Overall, I believe there is a lot of work that can be done into improving the glossary to account for these areas of confusion.
Unfortunately, we are limited on time so I am not sure how much can be incorporated into 1.0...

What do you think?

Reply:
If others agree about "download" being a generic term, then there's no problem. It's probably fine as is.